assets/helpers/user.js

```diff
@@ -1,12 +1,63 @@
 // @flow
```

Comment:
Yeah I thought about that. The only thing is, this file is 60 lines long, splitting it into three files of about 20 lines each felt like overkill. Is that something we'd want to do only when it starts to get longer?